### ISSUE TITLE
Notice: Undefined variable: qp in feedwordpress/syndicatedlink.class.php on line 747

### ISSUE THREAD

**Issue by zoul0813** [radgeek/feedwordpress#32]
When using "update checked" manually, I receive the following error repeated numerous times:

```
Notice: Undefined variable: qp in .../feedwordpress/syndicatedlink.class.php on line 747
```

I checked the code, and it appears the $gp is only defined inside of the conditional check 
```
// line 726
if (!is_null($uri) and strlen($uri) > 0 and $params['add_params']) :
```

$gp should be declared outside of the block, with a default value (null?) and then set inside this conditional to prevent this error.

This happens on one of the sites I use FWP on, but not another.

**Comment by hughiemolloy** [comment]
Same problem. I put 
$qp = null; 
before the if loop on line 726.

Can this be fixed in the next update?

**Comment by McBoof** [comment]
Oh pretty please include this in a release soon. FeedWordPress is the sweetest plugin ever, but this tiny buglet causes me and clients major upgrade headaches :-)

**Comment by radgeek** [comment]
Hey y'all,

Thanks for the heads-up on this issue. I have just pushed a fix for this to the main branch (7d3aeaed48e899fd0fa411ab222e8714d301cf1d), which you can get with a git pull. I will be putting out a release through the Subversion repository that encompasses this change probably later today.

Cheers,
-C
